modules/primer-marketing-buttons/README.md

```diff
@@ -0,0 +1,105 @@
+# Primer Marketing CSS Buttons
+
+[![npm version](http://img.shields.io/npm/v/primer-marketing-buttons.svg)](https://www.npmjs.org/package/primer-marketing-buttons)
+[![Build Status](https://travis-ci.org/primer/primer-css.svg?branch=master)](https://travis-ci.org/primer/primer-css)
+
+> Buttons are used for actions, like in forms, while textual hyperlinks are used for destinations, or moving from one page to another.
+
+This repository is a module of the full [primer-css][primer] repository.
+
+## Documentation
+
+<!-- %docs
+title: Marketing Buttons
+status: New Release
+-->
+
+The buttons on our marketing pages are based on Primer's core with the addition of more color options, color outline buttons, a transparent button, and larger button sizes.
+
+
+## Color
+
+The `btn` has been extended with the marketing color palette:
+
+```html
+
+<button class="btn btn-orange" type="button">Button button</button>
+<a class="btn btn-purple" href="#url" role="button">Link button</a>
+<button class="btn btn-blurple" type="button">Button button</button>
+
+```
+
+## Outline buttons
+
+The `btn-outline` has also been extended with the marketing color palette:
+
+```html
+
+<button class="btn btn-outline-orange" type="button">Button button</button>
+<a class="btn btn-outline-purple" href="#url" role="button">Link button</a>
+<button class="btn btn-outline-green" type="button">Button button</button>
+<div class="bg-gray-dark p-4 mt-4">
+  <button class="btn btn-transparent" type="button">
```

Comment:
`btn-transparent` shouldn't be grouped with the outline options, it should have is own description and be a separate example.